automatic/autohotkey.install/update.ps1

```diff
@@ -44,8 +44,9 @@ function global:au_BeforeUpdate {
 }
 
 function global:au_GetLatest {
-    $version = Invoke-WebRequest -Uri "$releases\version.txt" -UseBasicParsing | ForEach-Object Content
```

Comment:
Since this line, which uses `$releases` is being removed, should the definition of `$releases` on line 3, also be removed?

Reply:
Yes, of course it should. Not sure why my linter didn't pick that up. Fixed.